packages/model-viewer/src/features/scene-graph/api.ts

```diff
@@ -32,7 +32,7 @@ export declare interface ThreeDOMElementMap {
 }
 
 /** A 2D Cartesian coordinate */
-export interface Vector2 {
+export interface Vector2D {
```

Comment:
Also, I think you may have missed this one:

> Also check the members for compatibility with Vector2D.

I believe they're `u` and `v`, not `x` and `y`?

Reply:
Do we want that? I though the purpose of this Vector Interface was to not expose threejs' vector2, and ThreeJs Vector2 uses `x` and `y`. It seems to me that this interface and `Vector2D` in base have different purposes. wdyt?

Reply:
Exactly, we have zero interest in an API consistent with three.js (we're hiding their API). We want an API that is self-consistent and consistent with glTF. We already have an API called Vector2D that uses u and v, so using x and y here would be confusing. Besides, this feature is about textures, which are mapped in a u-v space.

Reply:
got it, sgtm